node/public.go

```diff
@@ -0,0 +1,66 @@
+// Package node provides a high level wrapper around tendermint services.
+package node
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/libs/log"
+	"github.com/tendermint/tendermint/libs/service"
+	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/privval"
+	"github.com/tendermint/tendermint/proxy"
+	"github.com/tendermint/tendermint/types"
+)
+
+// NewDefault constructs a tendermint node service for use in go process that
+// host their own process-local tendermint node. This is roughly
```

Comment:
Roughly equivalent? What do you mean by roughly? It is either used in process or not.

Reply:
removed roughly.